src/components/MDX/Challenges/Challenges.tsx

```diff
@@ -78,7 +78,7 @@ export function Challenges({
   children,
   isRecipes,
   noTitle,
-  titleText = isRecipes ? 'Try out some examples' : 'Try out some challenges',
+  titleText = isRecipes ? 'Try out some examples' : 'कुछ चुनौतियाँ आज़माएँ',
```

Comment:
This PR was only for preserving-and-resetting-state.md file.

Can you revert these changes